### PR TITLE
Send net-* flaky tests to #net-* channels instead of #networking.

### PR DESCRIPTION
**Pull request by tcnghia**
**What this PR does, why we need it**:

Send net-* flaky tests to #net-* channels instead of #networking.

See thread https://knative.slack.com/archives/CA9RHBGJX/p1587572619150500

### CODE REVIEW

File: tools/flaky-test-reporter/config/config.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 The Knative Authors
+x# Copyright 2019 The Knative Authors
```
Comment (chizhg) [inline tools/flaky-test-reporter/config/config.yaml:1]:
```suggestion
# Copyright 2019 The Knative Authors
```

File: tools/flaky-test-reporter/config/config.yaml [hunk 2]
```diff
@@ -24,49 +24,49 @@ jobConfigs:
     repo: serving
     type: postsubmit
     slackChannels:
-      - name: networking
+      - name: net-istio
         identity: CA9RHBGJX
```
Comment (chizhg) [inline tools/flaky-test-reporter/config/config.yaml:28]: Please also change the identity, which you can get by "right click the channel -> Additional options -> Copy link".

The same for other channels below as well.

Comment (tcnghia) [reply tools/flaky-test-reporter/config/config.yaml]: Done

**Comment by chizhg** [comment]
Thanks!
/lgtm
/approve

**Pull request merged**